bin/oref0-online.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
 echo; echo Starting oref0-online.
+# if we are connected to wifi but don't have an IP, try to get one
+if iwgetid -r wlan0 | egrep -q "[A-Z]|[a-z]|[0-9]?"; then
```

Comment:
The `egrep`  can be simplified and should allow all characters that are allowed in SSID's
I think you mean `+` (more than one)  instead of `?` (zero or 1) instance
For example I think it's cleaner to use `iwgetid -r wlan0 | egrep -q "[A-Za-z0-9_]+"`
Note: http://routersecurity.org/SSID.php says "Special characters (spaces, periods, dashes, underscores etc) are allowed. Its probably best to avoid the pipe character (vertical line). And, any use of a special character may be asking for trouble."